[2022-07-25 09:13:02,036] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dataquality_pipeline_v1.upload_to_s3_2019-01 manual__2022-07-25T09:12:48.190901+00:00 [queued]>
[2022-07-25 09:13:02,068] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dataquality_pipeline_v1.upload_to_s3_2019-01 manual__2022-07-25T09:12:48.190901+00:00 [queued]>
[2022-07-25 09:13:02,068] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:13:02,068] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-25 09:13:02,068] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:13:02,127] {taskinstance.py:1397} INFO - Executing <Task(LocalFilesystemToS3Operator): upload_to_s3_2019-01> on 2022-07-25 09:12:48.190901+00:00
[2022-07-25 09:13:02,144] {standard_task_runner.py:52} INFO - Started process 9860 to run task
[2022-07-25 09:13:02,163] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dataquality_pipeline_v1', 'upload_to_s3_2019-01', 'manual__2022-07-25T09:12:48.190901+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/dataquality_pipeline_v1.py', '--cfg-path', '/tmp/tmpreugg2lw', '--error-file', '/tmp/tmp25wkpvfd']
[2022-07-25 09:13:02,174] {standard_task_runner.py:80} INFO - Job 91: Subtask upload_to_s3_2019-01
[2022-07-25 09:13:02,347] {task_command.py:371} INFO - Running <TaskInstance: dataquality_pipeline_v1.upload_to_s3_2019-01 manual__2022-07-25T09:12:48.190901+00:00 [running]> on host d19767594dfa
[2022-07-25 09:13:02,591] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dataquality_pipeline_v1
AIRFLOW_CTX_TASK_ID=upload_to_s3_2019-01
AIRFLOW_CTX_EXECUTION_DATE=2022-07-25T09:12:48.190901+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-25T09:12:48.190901+00:00
[2022-07-25 09:13:02,623] {base.py:68} INFO - Using connection ID 'aws-conn' for task execution.
[2022-07-25 09:13:02,624] {base_aws.py:206} INFO - Credentials retrieved from login
[2022-07-25 09:13:04,669] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dataquality_pipeline_v1, task_id=upload_to_s3_2019-01, execution_date=20220725T091248, start_date=20220725T091302, end_date=20220725T091304
[2022-07-25 09:13:04,847] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-25 09:13:05,415] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
